_data/versions.yml

```diff
@@ -1942,70 +1942,53 @@ master:
-    - The Calico CNI plugin by default expects the /var/lib/calico/nodename file to be created by calico/node. To disable this feature, set `nodename_file_optional: true` in your CNI network configuration. [cni-plugin #480](https://github.com/projectcalico/cni-plugin/pull/480) (@caseydavenport)
-    - Fixes a bug where IPs could be assigned from disabled IP pools. [libcalico-go #806](https://github.com/projectcalico/libcalico-go/pull/806) (@ozdanborne)
-    - Fixes a bug where profiles were periodically and unnecessarily reprogrammed by kube-controllers. [libcalico-go #805](https://github.com/projectcalico/libcalico-go/pull/805) (@caseydavenport)
-    - Fixes a bug where nodes were periodically and unnecessarily processed by kube-controllers. [kube-controllers #216](https://github.com/projectcalico/kube-controllers/pull/216) (@caseydavenport)
-    - Closes a number of race conditions and failure scenarios in IPAM block allocation and releasing. [libcalico-go #785](https://github.com/projectcalico/libcalico-go/pull/785) (@caseydavenport)
-    - Improves log output around IPAM block allocation and releasing. [libcalico-go #785](https://github.com/projectcalico/libcalico-go/pull/785) (@caseydavenport)
-    - The self-hosted Kubernetes manifests now set mode 400 for TLS secrets by default [calico #1725](https://github.com/projectcalico/calico/pull/1725) (@caseydavenport)
-    - Fixes a rare bug where a node could in some circumstances advertise /26 blocks that it didn't own [calico #1712](https://github.com/projectcalico/calico/pull/1712) (@caseydavenport)
-    - Use networking.k8s.io api in place of deprecated extensions/v1beta1. [calico #1614](https://github.com/projectcalico/calico/pull/1614) (@bcreane)
-    - Fixes an interaction between failsafe inbound/outbound ports and do-not-track policy that resulted in failsafe ports being blocked if do-not-track policy was added. [felix #1718](https://github.com/projectcalico/felix/pull/1718) (@fasaxc)
-    - Fix bug in icmp validation where ipVersion was required for all icmp rules. [calicoctl #1814](https://github.com/projectcalico/calicoctl/pull/1814) (@ozdanborne)
-
-    #### Limitations
-
-    - **Offers only Kubernetes, OpenShift, OpenStack, and host endpoint integrations**: the
-    Mesos, DC/OS, rkt, and Docker orchestrators have not been tested
-    and are not supported. (Calico still supports Docker and rkt containers.)
-    We plan to resume support for the Mesos, DC/OS, and Docker
-    orchestrators in a future release.
-
-    - **GoBGP not supported**: Setting the `CALICO_NETWORKING_BACKEND` environment
-    variable to `gobgp` is not supported. See [Configuring calico/node](https://docs.projectcalico.org/v3.1/reference/node/configuration)
-    for more information. We plan to resume support for GoBPG in a future release.
-
-    - **Route reflectors cannot be clustered**: We plan to resume support for
-    this in a future release.
+- title: v3.1.0
+  components:
```

Comment:
I don't see networking-calico here yet.  (Probably because it was removed for 3.0 and so needs adding back now.)

I believe it will be:
```
    networking-calico:
      version: 3.1.0
      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=3.1.0
```

but will follow up to confirm once that actually exists (in about 1hr).